crates/nu-command/src/formats/to/toml.rs

```diff
@@ -51,7 +52,9 @@ fn helper(engine_state: &EngineState, v: &Value) -> Result<toml::Value, ShellErr
         Value::Int { val, .. } => toml::Value::Integer(*val),
         Value::Filesize { val, .. } => toml::Value::Integer(*val),
         Value::Duration { val, .. } => toml::Value::String(val.to_string()),
-        Value::Date { val, .. } => toml::Value::String(val.to_string()),
+        Value::Date { val, .. } => {
+            toml::Value::String(val.to_rfc3339_opts(SecondsFormat::AutoSi, false))
```

Comment:
I'm not familiar with `AutoSi`. Would you mind explaining it please?

Reply:
This will, by default include hours minutes and seconds and if there are milliseconds or microseconds and so on it will include those as well.

Reply:
Ah, ok. Thanks for explaining it.